server/controllers/auth/login.js

```diff
@@ -0,0 +1,85 @@
+// modules
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+
+const database = require("../../database");
+
+exports.login = (req, res, next) => {
+  // todo: 'remember me' feature
+  const email = req.body.email;
+  const password = req.body.password;
+
+  // find member using email address
+  database.query(`
+    SELECT
+      member_id, first_name, last_name, email_address, password, profile_picture, is_owner, is_moderator, is_blocked, created_at
+    FROM
+      member
+    WHERE
+      email_address = '${email}'
+  ;`).then(databaseResult => {
+    // get first member from array of objects
+    const memberData = databaseResult.rows[0];
+
+    // validate member password with encrypted password in database
+    bcrypt.compare(password, memberData.password).then(validatePassword => {
+      if (validatePassword) {
+
+        const loginToken = jwt.sign({
+          member_id: memberData.member_id,
+          email_address: memberData.email_address,
+          is_owner: memberData.is_owner,
+          is_moderator: memberData.is_moderator,
+          is_blocked: memberData.is_blocked,
+          created_at: memberData.created_at
```

Comment:
Is `created_at` necessary in token sign?

Reply:
Initially when I was working on the token payload, there was very less data for me to create the actual token, so I threw `created_at` 😅.

Reply:
lol. Put user firstName maybe, if that's still the case?

Reply:
> lol. Put user firstName maybe, if that's still the case?

Those are all null values. Didn’t you saw the database on testing the signup API?

Reply:
Oh, crap. You are right.